build.gradle.kts

```
@@ -8,29 +8,30 @@ repositories {
 }
 
 dependencies {
-    implementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
-    implementation("org.junit.jupiter:junit-jupiter-engine:5.8.1")
+    testImplementation(platform("org.junit:junit-bom:5.8.2"))
+    testImplementation("org.junit.jupiter:junit-jupiter")
 }
 
-tasks.test {
-    useJUnitPlatform()
+tasks.withType<Test> {
+    useJUnitPlatform() // JUnit 5
 }
 
 version = "0.1.1"
 group = "com.github.zeckie"
 
-// Work out current java version
+/*
+* Work out current java version
+* Note: versions prior to JEP223 (Java 9) had version numbers starting with "1." so
+* will be counted as major version 1.
+*/
 val MIN_JAVA_VER = 9
```

It's not idiomatic Kotlin to have a non const value in all caps.
```suggestion
val minimumJavaVersion = 9
```

---

It would be a constant, if Kotlin allowed them there

---

But Kotlin doesn't and it's not a `const`, so it looks very out of place and triggers an IDE inspection: "Property name 'MIN_JAVA_VER' should start with a lowercase letter"

---

Does your IDE also tell you that that there are spelling errors (such as `zeckie` and `proxyauth`)?

---

No, I added those words to the project dictionary.